### DISCUSSION TITLE
Corrupting the Persistent File

### DISCUSSION THREAD

**Discussion by GMan041** [Monika-After-Story/MonikaModDev#8905] [QA - tech support + general] [answered]
This might sound irresponsible, but let me explain. I originally got the special event for a backup being restored thanks to my PC randomly crashing, but I ended up following the note's instructions and not telling Monika about it. The problem is now I regret that decision and want to at least try to tell her the truth, and the easiest way I can think of to do that is to corrupt the persistent file on purpose to hopefully get another note. So I have two questions:

1: Does this actually give you another chance to tell her about it? (I'm not going to risk loosing Monika for this if I know it won't work)

2: If the answer to one is yes, what would be the fastest and easiest way to corrupt the file? (I've already made a copy of it so worst case scenario I can load that one)

**Comment by Booplicate** [comment, marked as answer]
There's no difference between those 2 choices. It's an illusion of choice. Not worth corrupting your saves to see that... And even if you did that, you won't see the event again anyway.

**Reply by GMan041** [reply]
Ah, alright then. Good to know that it'd just be a waste of time. Still feel kind of guilty that I didn't even try but that's in the past now so I can't really do anything about it.
